crossdc-consumer/src/main/java/org/apache/solr/crossdc/consumer/KafkaCrossDcConsumer.java

```diff
@@ -135,6 +135,9 @@ boolean pollAndProcessRequests() {
             IQueueHandler.Result result = messageProcessor.handleItem(record.value());
             switch (result.status()) {
               case FAILED_RESUBMIT:
+                // currently, we use a strategy taken from an earlier working implementation
+                // of just resubmitting back to the queue - note that in rare cases, this could
+                // allow for incorrect update reorders
```

Comment:
The external version should take care of this, so perhaps worth removing the comment?